modules/pubCommonId.js

```diff
@@ -9,7 +9,8 @@ import * as events from '../src/events.js';
 import CONSTANTS from '../src/constants.json';
 import { getStorageManager } from '../src/storageManager.js';
 
-const storage = getStorageManager();
+const MODULE_TYPE = 'fpid-module';
```

Comment:
can you also add this to the publisherprovidedid module?

Reply:
https://github.com/goldun/Prebid.js/blob/sharedid_vendorless_consent/modules/pubProvidedIdSystem.js

Reply:
looking at it again, this module doesnt seem to use storage manager, never mind